tests/ui/Viewer.js

```diff
@@ -2,14 +2,47 @@
  * @license Copyright (c) 2014-2016, CKSource - Frederico Knabben. All rights reserved.
  * For licensing, see LICENSE.md or http://ckeditor.com/license
  */
+/* bender-ckeditor-plugins: a11ychecker,toolbar,undo */
 /* bender-tags: a11ychecker,unit */
 
 ( function() {
 	'use strict';
 
-	bender.require( [ 'ui/Viewer', 'ui/ViewerDescription', 'mocking' ], function( Viewer, ViewerDescription, mocking ) {
+	bender.require( [ 'ui/Viewer', 'ui/ViewerDescription', 'mocking', 'EngineMock', 'testSuite', 'helpers/keyEvent' ], function( Viewer, ViewerDescription, mocking, EngineMock, testSuite, keyEvent ) {
+		testSuite.useEngine( EngineMock );
+
+		bender.editors = {
+			classic: {
+				name: 'editor1',
+				startupData: '<p>foo</p>'
+			},
+			inline: {
+				name: 'editor2',
+				creator: 'inline',
+				startupData: '<p>foo</p>'
+			}
+		};
+
+		var tests = {
+			tearDown: function() {
+				// For each test a11ychecker needs to be closed.
+				// Note that we have 2 editor instances, but only 1 can be enabled at
+				// a time.
+				function cleanupAC( editor ) {
+					var a11ychecker = editor._.a11ychecker;
+
+					if ( a11ychecker.issues && a11ychecker.issues.getFocused() ) {
```

Comment:
Do we need to reset the focus in this test suite? I don't think it's explicitly focused in any of these tests. With that `cleanupAC` might be removed and we can simply call `this.editors.classic._.a11ychecker.close()` and same for inline.